guide/ops/upgrades/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Upgrades
+---
```

Comment:
Front-matter is not needed anymore with gitbook

Reply:
@tbouron i simply moved `guide/ops/upgrade.md` to this file.  useful comment but rather than block this PR because of a piecemeal improvement to existing content probably we should merge and have another PR remove unnecessary front-matter from all files?